pkg/controller/plan/util/openstack.go

```diff
@@ -9,11 +9,12 @@ import (
 	"github.com/konveyor/forklift-controller/pkg/controller/provider/web/openstack"
 )
 
-func OpenstackVolumePopulator(image *openstack.Image, sourceUrl *url.URL, targetNamespace, secretName, migrationName string) *api.OpenstackVolumePopulator {
+func OpenstackVolumePopulator(image *openstack.Image, sourceUrl *url.URL, targetNamespace, secretName, vmId string) *api.OpenstackVolumePopulator {
 	return &api.OpenstackVolumePopulator{
 		ObjectMeta: meta.ObjectMeta{
 			Name:      image.Name,
 			Namespace: targetNamespace,
+			Labels:    map[string]string{"vmID": vmId},
```

Comment:
the commit message says we use the migration name in the label to make it simpler to correlate - and here we set the vmId even though the VM doesn't exist at this point, so why not having the migration in-addition/instead?